server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiExceptionHandler.java

```diff
@@ -64,8 +64,7 @@
                     (ctx, req, cause) -> newResponse(ctx, HttpStatus.NOT_FOUND, cause,
                                                      "Entry '%s' does not exist.", cause.getMessage()))
                .put(EntryNoContentException.class,
-                    (ctx, req, cause) -> newResponse(ctx, HttpStatus.NO_CONTENT, cause,
-                                                     "Entry '%s' does not have content.", cause.getMessage()))
+                    (ctx, req, cause) -> newResponse(ctx, HttpStatus.NO_CONTENT, cause))
```

Comment:
HttpResponse.of(HttpStatus.NO_CONTENT) would be enough?

Reply:
Thank you for review. Fixed in 9e869bc